client-s3.go

```diff
@@ -678,6 +678,19 @@ func (c *s3Client) listInRoutine(contentCh chan *clientContent) {
 	}
 }
 
+// S3 offers a range of storage classes designed for
+// different use cases, following list captures these.
+const (
+	// General purpose.
+	// s3StandardStorageClass = "STANDARD"
+	// Infrequent access.
+	// s3InfrequentStorageClass = "STANDARD_IA"
+	// Reduced redundancy access.
+	// s3RedundancyStorageClass = "REDUCED_REDUNDANCY"
+	// Archive access.
+	s3GlacierStorageClass = "GLACIER"
```

Comment:
Rename it to s3StorageClassGlacier. Variable component comes at the end. This will also allow IDEs to autocomplete.